docs/quickstarts/ansible-related-documentation/ansible-related-documentation.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: Quickstarts
```

Comment:
So I'm not sure what's up here and why `Quickstarts` will work here, but not in the metadata files. The validator seems very picky about it. I'd match all these to `QuickStarts`